clients/src/main/resources/common/message/ConsumerGroupHeartbeatRequest.json

```diff
@@ -35,6 +38,8 @@
       "about": "-1 if it didn't change since the last heartbeat; the maximum time in milliseconds that the coordinator will wait on the member to revoke its partitions otherwise." },
     { "name": "SubscribedTopicNames", "type": "[]string", "versions": "0+", "nullableVersions": "0+", "default": "null", "entityType": "topicName",
       "about": "null if it didn't change since the last heartbeat; the subscribed topic names otherwise." },
+    { "name": "SubscribedTopicRegex", "type": "string", "versions": "1+", "nullableVersions": "1+", "default": "null",
+      "about": "null if it didn't change since the last heartbeat; the subscribed topic regex otherwise" },
```

Comment:
Didn't we say we want to commit this change in a separate PR? 
https://github.com/apache/kafka/pull/15188#issuecomment-1918915086

Reply:
Will change this to a separate PR